src/app/tests/suites/certification/Test_TC_CC_3_3.yaml

```diff
@@ -87,14 +98,24 @@ tests:
               - name: "optionsOverride"
                 value: 0
 
+    - label: "Wait 25ms"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 25
+
     - label:
           "Check current hue attribute value matched the value sent by the last
           command"
-      disabled: true
       command: "readAttribute"
       attribute: "current hue"
       response:
-          value: -1
+          constraints:
+              type: uint8
+              minValue: 0
+              maxValue: 254
```

Comment:
Again, this is not doing what the test label says it should be doing.

Reply:
Modified the label as per testplan